apps/web/components/ui/Dropdown.tsx

```diff
@@ -25,8 +25,11 @@ export const DropdownMenuContent = forwardRef<HTMLDivElement, DropdownMenuConten
   ({ children, ...props }, forwardedRef) => {
     return (
       <DropdownMenuPrimitive.Content
+        portalled={false}
```

Comment:
As explained, this makes the dropdown be added in the `body` tag when "portalled".

Reply:
I'm not 100% sold on the nomenclature. Since is a boolean this could be `isPortal` to improve legibility.